app/Http/Controllers/Manager/WalletController.php

```diff
@@ -328,6 +329,108 @@ public function withdraw(Request $request, AdsRpcClient $rpcClient): JsonRespons
         return $this->withdrawAds($request, $rpcClient);
     }
 
+    private function getSwashBSCAddress(): WalletAddress
```

Comment:
I can't understand logic of this code, if it is a bad config you should raise error on running.
It is not a good time to validate swash bsc address on each request.

Reply:
It's the first place this config is loaded.